docs/docs/integrating-with-other-libraries.md

```diff
@@ -255,16 +278,15 @@ A copy is made of the model's attributes to form the initial state. Every time t
 To demonstrate its use, we will use a basic input wrapper. The component will render the `'text'` attribute of the provided model, calling an update function whenever the input is changed, presumably to update the model.
 
 ```js
-function Input({ text, handleChange }) {
-  return <input value={text} onChange={handleChange} />;
+function Input(props) {
+  return <input value={props.text} onChange={props.handleChange} />;
 }
 
-function BackboneModelAdapterDemo() {
-  const model = new Backbone.Model({ text: 'Sam' });
-  const handleChange = event => model.set('text', event.target.value);
-  const WrappedInput = backboneModelAdapter(Input);
-  return <WrappedInput model={model} handleChange={handleChange} />;
-}
+const WrappedInput = connectToBackboneModel('Input');
+const model = new Backbone.Model({ text: 'Sam' }); 
+const handleChange = event => model.set('text', event.target.value);
+
+<WrappedInput model={model} handleChange={handleChange} />
```

Comment:
Can you still provide a whole example here? It’s fine to have a `Demo` component rendering `WrappedInput`. The only essential part is `WrappedInput` assignment should happen outside that component. (And you’d need to update the codepen too).